pkg/cli/admin/release/git_test.go

```diff
@@ -0,0 +1,135 @@
+		wantErr bool
+	}{
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1eBug 1743564: test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{1743564}, Subject: "test",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1eBug 1743564: test [trailing]",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{1743564}, Subject: "test [trailing]",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1e[release-4.1] Bug 1743564: test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{1743564}, Subject: "test",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1e [release-4.1] Bug 1743564: test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{1743564}, Subject: "test",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1e [release-4.1] Bug 1743564 : test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{1743564}, Subject: "test",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1e [release-4.1] Bugs 1743564 : test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{1743564}, Subject: "test",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1e [release-4.1] Bugs 1743564,: test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{1743564}, Subject: "test",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1e [release-4.1] Bugs , 17 43,564,: test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{17, 43, 564}, Subject: "test",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1e [release-4.1] bugs , 17 43,564,: test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: []int{17, 43, 564}, Subject: "test",
+				},
+			},
+		},
+		{
+			input: "abc\x1e1\x1eMerge pull request #145 from\x1e [release 4.1] bugs , 17 43,564,: test",
+			want: []MergeCommit{
+				{
+					ParentCommits: []string{}, Commit: "abc", PullRequest: 145, CommitDate: time.Unix(1, 0).UTC(),
+					Bugs: nil, Subject: "[release 4.1] bugs , 17 43,564,: test",
```

Comment:
don't we want `[release 4.1]` striped? is the regex intentionally missing the space?

Reply:
Yes, we decided not to strip those because it gets much more complex.